src/components/parser/log.ts

```diff
@@ -158,7 +158,7 @@ export class Parser {
             }
             // append the read line, since we have a corresponding result in the making
             if (searchesEmptyLine) {
-                if (line.trim() === '' || (insideError && line.match(/^\s/))) {
```

Comment:
I don't think we want to remove `(insideError && line.match(/^\s/))` here.

The point of removing it would be to test the line against the `messageLine` regex but anything that matches `^\s` will never match `messageLine` (because `messageLine` starts with `^l`).